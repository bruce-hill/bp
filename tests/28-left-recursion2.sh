# Left recursion has some tricky edge cases like this:
bp -p 'foo: (foo / `a-z) (foo / `a-z) `!; foo => "{@0}"'
