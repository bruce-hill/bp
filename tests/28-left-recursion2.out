{ab!}
{ab!c!}
{ab!cd!!}
